### DISCUSSION TITLE
Config to disable AbyssalCraft Plague Potion Clouds

### DISCUSSION THREAD

**Discussion by WaitingIdly** [ACGaming/UniversalTweaks#417] [Tweak Suggestions]
Some AbyssalCraft entities spawn in a lingering potion effect when they die, similar to how creepers with a potion effect do when they explode. Additionally, any entity with either the `coralium_plague` or `dread_plague` effect will also spawn in a potion cloud.
This is a neat cosmetic thing, but if you have a mob farm killing off these mobs, it can be incredibly laggy, since once a single mob generates a cloud, every mob killed will generate a cloud.
A config to disable generating this potion effect, via a mixin of `PlagueEventHandler#onDeath`, would be a reasonable solution for a pack developer who intends to have these mobs killed in a mob farm, or for a player who is getting significant lag due to the outlined situation.

**Comment by ACGaming** [comment]
Implemented in 7ca6614294329f94b8b3cb74dd2075b6524835f8
